common/src/main/java/tachyon/Constants.java

```diff
@@ -200,18 +201,18 @@
   public static final String WORKER_NETWORK_NETTY_SHUTDOWN_TIMEOUT =
       "tachyon.worker.network.netty.shutdown.timeout";
   public static final String WORKER_ALLOCATE_STRATEGY_CLASS =
-      "tachyon.worker.allocate.strategy.class";
-  public static final String WORKER_EVICT_STRATEGY_CLASS = "tachyon.worker.evict.strategy.class";
+      "tachyon.worker.allocator.class";
```

Comment:
yes, that will be done in another follow PR because these constants are used everywhere in the codebase